kolibri/core/auth/api_urls.py

```diff
@@ -42,6 +43,11 @@
             r"^setnonspecifiedpassword$",
             SetNonSpecifiedPasswordView.as_view(),
             name="setnonspecifiedpassword",
-        )
+        ),
+        url(
+            r"^usernameexists",
```

Comment:
```suggestion
            r"^usernameexists$",
```

So it matches the URL exactly. It's not going to happen in practice, but sending the request to `/api/auth/usernameexistssss` will still get resolved to this view.